docs/isa-support.rst

```diff
@@ -123,32 +123,32 @@ opcode  src   imm   description                                          PREVAIL
 0xbd    any   0x00  if dst <= src goto +offset                              Y      Y      Y    jle-reg
 0xbe    any   0x00  if (uint32_t)dst <= (uint32_t)src goto +offset          Y      Y      Y    jle32-reg
 0xbf    any   0x00  dst = src                                               Y      Y      Y    ldxb-all
-0xc3    any   0x00  lock \*(uint32_t \*)(dst + offset) += src               no     no     no   lock_add32
-0xc3    any   0x01  lock::                                                  no     no     no   lock_fetch_add32
+0xc3    any   0x00  lock \*(uint32_t \*)(dst + offset) += src               Y      Y      Y    lock_add32
```

Comment:
Only the bpf2c column should be updated, since PREVAIL and uBPF do not support these instructions.